testsuite/features/step_definitions/common_steps.rb

```diff
@@ -35,10 +35,19 @@
 
   raise 'SHA256 checksum validation failed' unless validate_checksum_with_file(original_iso_name, iso_path, checksum_path)
 
-  mount_point = "/srv/www/htdocs/#{name}"
-  get_target('server').run("mkdir -p #{mount_point}")
-  get_target('server').run("grep #{iso_path} /etc/fstab || echo '#{iso_path}  #{mount_point}  iso9660  loop,ro,_netdev  0 0' >> /etc/fstab")
-  get_target('server').run("umount #{iso_path}; mount #{iso_path}")
+  if $is_container_provider
+    mount_point = '/srv/www/distributions'
+    get_target('server').run("mkdir -p #{mount_point}")
+    # this needs to be run outside the container
+    get_target('server').run_local("mgradm distro copy #{iso_path} #{name}", verbose: true)
+    get_target('server').run("ln -s #{mount_point}/#{name} /srv/www/htdocs/pub/")
+  else
+    mount_point = "/srv/www/htdocs/#{name}"
```

Comment:
Double checking: Is this right or should it be **/srv/www/htdocs/pub/**  ?

Reply:
Nice catch. I simply took it from the old code. However, since I changed

```diff
-    And I enter "http://127.0.0.1/centos-7-iso" as "url"
+    And I enter "file:///srv/www/htdocs/pub/centos-7-iso" as "url"
```
I need to adjust the `mount_point` path, too.

Reply:
Fixed.